specifications/api/swagger.yaml

```diff
@@ -321,6 +321,57 @@ x-poolMetadataSource: &poolMetadataSource
       format: uri
       example: https://smash.cardano-mainnet.iohk.io/
 
+x-credentialPubKey: &credentialPubKey
+  description: A public key (public key without chain code) for credential - 32 bytes
+  type: string
+  format: bech32
+  pattern: "^((addr_vk)|(stake_vk))1[0-9a-z]*$"
+  example:
+    0: addr_vk13pfp60uf4ru7cmj5d57xjdzj6lagn328ah67h35urqrl92jn094z0s07fcy2pvfw3xfhna6fa24na8nk6lraymjcrl5wwj6chsjqtjqlfw44z
+    1: stake_vk13pfp60uf4ru7cmj5d57xjdzj6lagn328ah67h35urqrl92jn094z0s07fcy2pvfw3xfhna6fa24na8nk6lraymjcrl5wwj6chsjqtjqlfw44z
+
+x-anyAddress: &anyAddress
+  description: A Shelley address representing either enterprise, reward account or delegating address
+  type: string
+  format: bech32
+  pattern: "^((addr)|(stake)|(addr_test)|(stake_test))1[0-9a-z]*$"
+  example:
+    0: addr13pfp60uf4ru7cmj5d57xjdzj6lagn328ah67h35urqrl92jn094z0s07fcy2pvfw3xfhna6fa24na8nk6lraymjcrl5wwj6chsjqtjqlfw44z
+    1: stake_test13pfp60uf4ru7cmj5d57xjdzj6lagn328ah67h35urqrl92jn094z0s07fcy2pvfw3xfhna6fa24na8nk6lraymjcrl5wwj6chsjqtjqlfw44z
+
+x-credential: &credential
+  nullable: false
+  additionalProperties:
+    $ref: "#/components/schemas/CredentialValue"
+
+x-script: &script
+  description: |
+    Script allows establishing either spending or staking guard that could be removed
+    when a special combination of signing keys is present.
+
+    The script can be constructed using four primitives:
+     - verification key hash for which a corresponding signing key is expected
+     - all condition for which all list elements' expectation are met for the condition to be valid
+     - any condition for which any list element' expectation is to be met for the condition to be valid
+     - some condition for which a minimal number of list elements' expectation is to be met for the condition to be valid
+    example:
+      - "script_vkh18srsxr3khll7vl3w9mqfu55n6wzxxlxj7qzr2mhnyreluzt36ms"
+      - { "all": [ "script_vkh18srsxr3khll7vl3w9mqfu55n6wzxxlxj7qzr2mhnyreluzt36ms"
+                  , "script_vkh18srsxr3khll7vl3w9mqfu55n6wzxxlxj7qzr2mhnyreluzt37ms"] }
+      - { "all": [ "script_vkh18srsxr3khll7vl3w9mqfu55n6wzxxlxj7qzr2mhnyreluzt36ms"
+                  , { "any" : [ "script_vkh18srsxr3khll7vl3w9mqfu55n6wzxxlxj7qzr2mhnyreluzt37ms"
+                              , "script_vkh18srsxr3khll7vl3w9mqfu55n6wzxxlxj7qzr2mhnyreluzt38ms"] }
+                  ] }
+      - { "all": [ "script_vkh18srsxr3khll7vl3w9mqfu55n6wzxxlxj7qzr2mhnyreluzt36ms"
+                  , { "some" : { "from" : [ "script_vkh18srsxr3khll7vl3w9mqfu55n6wzxxlxj7qzr2mhnyreluzt37ms"
+                                          , "script_vkh18srsxr3khll7vl3w9mqfu55n6wzxxlxj7qzr2mhnyreluzt38ms"]
+                               , "at_least" : 2
+                               }
+                    } ] }
```

Comment:
There seems to be some formatting issue here:

![Screenshot from 2020-10-30 15-37-19](https://user-images.githubusercontent.com/42900201/97718203-f4a0f200-1ac5-11eb-82c4-d6eaf752a876.png)

Reply:
thanks, will fix